Marlin/Configuration_adv.h

```diff
@@ -801,10 +801,10 @@
 //#define ASSISTED_TRAMMING
 #if ENABLED(ASSISTED_TRAMMING)
 
-  // Define positions for probing points, use the hotend as reference not the sensor.
-  #define TRAMMING_POINT_XY { {  20, 20 }, { 200,  20 }, { 200, 200 }, { 20, 200 } }
+  // Define positions for probe points.
+  #define TRAMMING_POINT_XY { {  20, 20 }, { 180,  20 }, { 180, 180 }, { 20, 180 } }
```

Comment:
instead of 180, this should not be a offset from BED_SIZE, like `X_BED_SIZE - 20`?

Reply:
Or better yet, Use `PROBING_MARGIN` instead of an arbitrary number.

Reply:
I’m not trying to make these auto-select. The original values simply could not work with other defaults.

These really need to factor the probe offset (not just the margin), but the offset is not accessible in a macro, only in a constexpr or at runtime.